app/src/pages/inside/stepPage/stepPage.jsx

```diff
@@ -299,6 +292,18 @@ export class StepPage extends Component {
     }
   };
 
+  isGroupOperationAvailable = (editedData) => {
```

Comment:
Let's rename it to `isDefectGroupOperationAvailable`. This will help to avoid confusion with group operations available from the "Actions" menu.